runtime/native.go

```diff
@@ -348,6 +369,19 @@ func getNativeType(rtype reflect.Type) *Type {
 	return t
 }
 
+func newNativeField(name string, i, refDepth int) *Object {
+	// Closes over `i` and `refDepth`
+	nativeFieldGet := func(f *Frame, desc, instance *Object, owner *Type) (*Object, *BaseException) {
+		v := toNativeUnsafe(instance).value
+		for i := 0; i < refDepth; i++ {
+			v = v.Elem()
+		}
+		return WrapNative(f, v.Field(i))
+	}
+
+	return &Object{typ: &Type{slots: typeSlots{Get: &getSlot{nativeFieldGet}}}}
```

Comment:
The property has an fget attribute which somebody in Python land could call with any old parameters (I think fget is not yet exposed by Grumpy, but it will be eventually).